articles/virtual-network/nat-gateway/faq.yml

```diff
@@ -90,7 +90,11 @@ sections:
       - question: Can I use Virtual Network NAT gateway with Azure Firewall?
         answer: |
           Yes. For more information about Virtual Network NAT integration with Azure Firewall, see [Scale SNAT ports with Azure NAT Gateway](../../firewall/integrate-with-nat-gateway.md).
-          
+
+      - question: Can I use Virtual Network NAT gateway with Service Endpoints?
+        answer: | 
+           Yes. The addition of a Virtual Network NAT Gateway to a subnet with Service Endpoints does not affect the Service Endpoints. Service Endpoints enable a more specific route for the destination Azure service traffic they represent. Traffic for the Service Endpoint will continue to be routed toward the service and wont go via the NAT Gateway.
```

Comment:
@bengimblett could we also maybe add a link to the service endpoint doc so that customers can read more there if they  want? 

Something like:
To learn more about service endpoints, see Service Endpoint Overview. 

And link to this doc here: https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview

Reply:
updated :)